travis/install_protobuf.sh

```diff
@@ -11,9 +11,9 @@ if [ "$protobuf_dist" != "" ]; then
   cd $protobuf_dist
 fi
 
-wget https://github.com/google/protobuf/archive/v3.0.0-beta-1.tar.gz
-tar -xvzf v3.0.0-beta-1.tar.gz
-cd protobuf-3.0.0-beta-1
+wget https://github.com/google/protobuf/archive/v3.0.0-beta-2.tar.gz
```

Comment:
I like the idea of updating install_protobuf.sh to use beta-2 on OS X. Would you like to make a PR for that? If so, please include a comment that explains that OS X requires beta-2, but grpc doesn't build with beta-2 yet on Linux.

I'm guessing that the reason grpc builds with protobuf beta-2 on OS X is that the build skips the C# parts on that platform.